packages/orbit-components/src/Heading/README.md

```diff
@@ -44,6 +44,7 @@ Table below contains all types of the props available in Heading component.
 | `"h5"`  | `"title3"`          |           |
 | `"h6"`  | `"title4"`          |           |
 | `"div"` | `"title5"`          |           |
+| `"div"` | `"title6"`          |           |
```

Comment:
```suggestion
|   | `"title6"`          |           |
```

Please do this change manually, as accepting the suggestion here will leave the filme unformatted 🤞